README.md

```diff
@@ -97,13 +104,15 @@ This will load the various `.env` files for Algorand, Circle, etc. credentials -
 most other environment variables will be overridden in favor of those specified
 in the `docker-compose.yml` file.
 
-Next, the CMS key needs to be added to the admin user created by Directus.
+### Adding the CMS key to the admin user
+
+The CMS key needs to be added to the admin user created by Directus.
 Otherwise, the API cannot authenticate even though its CMS key matches.
 
 - Visit http://localhost:3001
-- Authenticate with the CMS [admin email & password](docker-compose.yml#L77)
+- Authenticate with the CMS admin email & password
 - Go to the [user directory](http://localhost:3001/admin/users)
-- Click the admin user and enter the [CMS key](docker-compose.yml#L70) into the "Token" field
+- Click the admin user and enter the CMS key into the "Token" field
```

Comment:
Didn't want to point to docker-compose stuff if a developer isn't using it; seemed potentially confusing.